src/esp/physics/CollisionGroupHelper.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "CollisionGroupHelper.h"
+
+namespace esp {
+namespace physics {
+
+// initialize the default collision group names
+std::map<std::string, CollisionGroup>
+    CollisionGroupHelper::collisionGroupNames = {
+        {"Default", CollisionGroup::Default},
```

Comment:
Optional: This makes it clear this could be a Corrade EnumSet https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html with all the convience functions you get from that.